src/libraries/System.Formats.Tar/tests/System.Formats.Tar.Tests.csproj

```
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <StringResourcesPath>$(LibrariesProjectRoot)/Common/tests/Resources/Strings.resx</StringResourcesPath>
+    <StringResourcesPath>$(MSBuildProjectDirectory)\..\src\Resources\Strings.resx</StringResourcesPath>
```

is this change intended?

---

Yes, for SR.TarExtractingResultsLinkOutside.

---

No other test project uses `MSBuildProjectDirectory`. Can't you just use a relative path? 
```suggestion
    <StringResourcesPath>..\src\Resources\Strings.resx</StringResourcesPath>
```

That's what we do in other projects.

---

> Can't you just use 

You are asking a different question now. Not sure why it is bothering you this much, it is 100% same thing!

---

I didn't know it was the same thing. In that case, is fine.